scvi/dataset/_anndata.py

```diff
@@ -32,7 +32,7 @@ def get_from_registry(adata: anndata.AnnData, key: str) -> np.array:
 
     Returns
     -------
-    np.array containing the data requested
+    The requested data
```

Comment:
what type is it?

Reply:
the type of the function signature gets placed in the doc